les/player-v1.jsonld

```diff
@@ -0,0 +1,22 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "@protected": true,
+    "pl": "https://sphereon-opensource.github.io/vc-contexts/les/player-v1#",
+    "schema": "http://schema.org/",
+    "player": {
+      "@id": "https://sphereon-opensource.github.io/vc-contexts/les/player-v1#player",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "playerId": "pl:playerId",
+        "name": "pl:name",
```

Comment:
Suggest to rename to firstName to make it more clear

Reply:
Done